plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/util/DataSkippingStatsColumnsUtils.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+import io.trino.spi.TrinoException;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+import static io.trino.plugin.deltalake.DeltaLakeErrorCode.DELTA_LAKE_INVALID_SCHEMA;
+import static io.trino.plugin.deltalake.transactionlog.DeltaLakeSchemaSupport.DATA_SKIP_STATS_COLUMN_CONFIGURATION_KEY;
+import static java.lang.String.format;
+
+public final class DataSkippingStatsColumnsUtils
+{
+    private static final String SPECIAL_CHARS = "!@#$%^&*()+-={}|[]:\\\";'<>,.?/";
+    private static final Escaper ESCAPER;
+
+    static {
+        Escapers.Builder builder = Escapers.builder();
+        for (char c : SPECIAL_CHARS.toCharArray()) {
+            builder.addEscape(c, "\\" + c);
+        }
+        ESCAPER = builder.build();
+    }
+
+    private DataSkippingStatsColumnsUtils() {}
+
+    // convert Trino names to Delta data_skipping_stats_columns property
+    public static String toDataSkippingStatsColumnsString(Iterable<String> dataSkippingStatsColumns)
+    {
+        ImmutableSet.Builder<String> result = ImmutableSet.builder();
+        for (String column : dataSkippingStatsColumns) {
+            result.add(toDeltaName(column));
+        }
+        return Joiner.on(",").join(result.build());
+    }
+
+    // parse Delta data_skipping_stats_columns property to Trino names
+    public static Set<String> getDataSkippingStatsColumns(Optional<String> dataSkippingStatsColumnsProperty)
+    {
+        if (dataSkippingStatsColumnsProperty.isEmpty()) {
+            return ImmutableSet.of();
+        }
+
+        String property = dataSkippingStatsColumnsProperty.get();
+        StringBuilder current = new StringBuilder();
+        boolean inBackticks = false;
+
+        ImmutableSet.Builder<String> result = ImmutableSet.builder();
+        List<String> parts = new ArrayList<>();
+        Set<String> seen = new HashSet<>();
+        for (char c : property.toCharArray()) {
+            if (c == '`') {
```

Comment:
@ebyhr Is it ok for you we are following the delta using ` to quote the special column?
Do you have other thoughts?

Reply:
Let's follow the Delta's quote rule.